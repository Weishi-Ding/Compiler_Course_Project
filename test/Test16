/*Test class array*/
class Integer{
    int i;
    constructor Integer(int i){

    }

    constructor Integer(){

    }
}

Integer[] initArray(){
    Integer[] array : = new Integer[10];
    int i;
    for(i = 0; i < 10; i = i+1){
        array[i] : = new Integer(i);
    } 

    return array;
}

int main() {
    Integer[] array : = initArray();

    int i;
    for(i = 0; i < 10; i = i+1){
        print(array[i].i);
    } 
}
