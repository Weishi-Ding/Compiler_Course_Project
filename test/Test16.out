3
1
10
20
"public static"
"public"
